### PR TITLE
Fix search build (missing repo resource)

### PR DESCRIPTION
**Pull request by mitchdenny**
This PR adds ```azure-sdk-tools``` to the repo resource list for the search pipeline (was missing).

### CODE REVIEW

**Comment by mitchdenny** [comment]
Merging because this won't pass any CI builds.

File: sdk/search/ci.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ resources:
     - repository: azure-sdk-build-tools
       type: github
       name: internal/azure-sdk-build-tools
+    - repository: azure-sdk-tools
```
Comment (weshaggard) [inline sdk/search/ci.yml]: @chidozieononiwu looks like another yml file missed. please double check the rest.

Comment (chidozieononiwu) [reply sdk/search/ci.yml]: After a quick [search](https://github.com/Azure/azure-sdk-for-java/search?p=1&q=repository%3A+azure-sdk-build-tools&unscoped_q=repository%3A+azure-sdk-build-tools) I can confirm that the azure-sdk-tools repository is present on all 20 ci.yml files. Apologies for the earlier missing ones.

**Pull request merged**